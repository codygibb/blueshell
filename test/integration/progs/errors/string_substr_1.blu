s := "hello world"
print s.substr(0, 0) # should be valid
print s.substr(3, 0)
