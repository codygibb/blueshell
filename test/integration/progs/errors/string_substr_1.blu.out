
error: illegal argument: start index must be < stop index
